tools/gbench/report.py

```diff
@@ -143,32 +146,43 @@ def extract_field(partition, field_name):
     return [lhs, rhs]
 
 
-def print_utest(partition, utest_alpha, first_col_width, use_color=True):
-    timings_time = extract_field(partition, 'real_time')
-    timings_cpu = extract_field(partition, 'cpu_time')
-
+def calc_utest(timings_cpu, timings_time):
     min_rep_cnt = min(len(timings_time[0]),
                       len(timings_time[1]),
                       len(timings_cpu[0]),
                       len(timings_cpu[1]))
 
     # Does *everything* has at least UTEST_MIN_REPETITIONS repetitions?
     if min_rep_cnt < UTEST_MIN_REPETITIONS:
-        return []
-
-    def get_utest_color(pval):
-        return BC_FAIL if pval >= utest_alpha else BC_OKGREEN
+        return False, None, None
 
     time_pvalue = mannwhitneyu(
         timings_time[0], timings_time[1], alternative='two-sided').pvalue
     cpu_pvalue = mannwhitneyu(
         timings_cpu[0], timings_cpu[1], alternative='two-sided').pvalue
 
+    return (min_rep_cnt >= UTEST_OPTIMAL_REPETITIONS), cpu_pvalue, time_pvalue
+
+
+def print_utest(partition, utest_alpha, first_col_width, use_color=True):
+    def get_utest_color(pval):
+        return BC_FAIL if pval >= utest_alpha else BC_OKGREEN
+
+    timings_time = extract_field(partition, 'real_time')
+    timings_cpu = extract_field(partition, 'cpu_time')
+    stable_results, cpu_pvalue, time_pvalue = calc_utest(
```

Comment:
What does `stable_results` mean here? Is this about the pvalue? Or repetition count?
Please s/`stable_results`/`sufficient_repetitions`/

Reply:
I had it named `optimal_repetitions` previously but changed it to `stable_results` because I thought that communicating that result**s** (both pvalues) not being stable was sufficient (generic enough) to indicate the erroneous state. That way we are avoiding leaking details of how this error condition is actually evaluated by the underlying function call: `calc_utest`.

Let me know what you think about that.